witness/tests/Test/Dscp/Witness/Block/Block.hs

```diff
@@ -108,7 +108,7 @@ spec = describe "Block validation + application" $ do
         lift . throwsSome $ submitBlock badBlock
 
     it "Wrong difficulty is fatal" $ witnessProperty $ do
-        _ <- stop $ pendingWith "To be resolved in [DSCP-261]"
+        _ <- stop ()  -- To be resolved in [DSCP-261]
```

Comment:
Why? It's still `hspec`, it should still work, shouldn't it?

Reply:
Tasty treats test outcomes as binary values, `pending` should be mapped to either `failure` (default) or `success`. I didn't find an easy way to make them map to `success`, thus did this change.

Reply:
@Martoon-00 https://hackage.haskell.org/package/tasty-hspec-1.1.5/docs/Test-Tasty-Hspec.html#t:TreatPendingAs
and I'm sure that with little effort you'll find the way to set it in the code, not via command-line flag

Reply:
Ok, I fixed it by passing that CLI option to the discovery command in `tests/Main.hs`. It could be passed in code, but probably only when tasty is used as DSL.